src/algorithms/crypto/vrf.go

```diff
@@ -20,9 +23,11 @@ func (self *VRFResult_I) MaxValue() util.Bytes {
 func (self *VRFKeyPair_I) Generate(input util.Bytes) VRFResult {
 	// sig := new(BLS).Sign(input, self.SecretKey)
 	var blsSig util.Bytes
+
+	digest := blake2b.Sum256(blsSig)
 	ret := &VRFResult_I{
-		Proof_: blsSig,
-		Digest_: blake2b.Sum256(blsSig)
```

Comment:
Missing `,` and different type. `blake2b.Sum256` returns `[32]byte`, and `Digest_` expects `[]byte`.